src/Mutator/Util/MutatorProfile.php

```diff
@@ -232,6 +232,10 @@ final class MutatorProfile
         Mutator\Unwrap\UnwrapUcWords::class,
     ];
 
+    public const EXTENSIONS = [
+        Mutator\Extensions\BCMath::class,
+    ];
+
     public const DEFAULT = [
```

Comment:
Shouldn't `@extensions` be added here to the default profile? As it is done for `mbstring` extension?

https://github.com/infection/infection/pull/664/files#diff-071045e7645d747e46b60edd948a3537R254

Reply:
fixed by merging with master